### ISSUE TITLE
User Wizard creator must use the Identity plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#245]
```
Actually, this wizard use its own UI for identity and address, we must re-use 
coded features and use Identity::IdenityEditorWidget instead.

```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 17 Dec 2012 at 9:14

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision cec7f6afee57.

Identity widget manages login/password
```

Original comment by `eric.mae...@gmail.com` on 18 Dec 2012 at 8:27

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 464b46549a29.

Using Identity plugin in UserCreatorWizard. Address & contact still needs a 
refactoring
```

Original comment by `eric.mae...@gmail.com` on 18 Dec 2012 at 8:27

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 2877365b007e.

Adding properties for the use inside wizard fields
```

Original comment by `eric.mae...@gmail.com` on 19 Dec 2012 at 9:39

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 96ebd351b64c.
```

Original comment by `eric.mae...@gmail.com` on 19 Dec 2012 at 9:39
- Changed state: **Fixed**
